vscode/microsoft-kiota/src/steps.ts

```diff
@@ -148,7 +149,8 @@ export function transformToGenerationconfig(deepLinkParams: Record<string, strin
     return generationConfig;
 }
 
-export async function generateSteps(existingConfiguration: Partial<GenerateState>, languagesInformation?: LanguagesInformation) {
+
+export async function generateSteps(existingConfiguration: Partial<GenerateState>, languagesInformation?: LanguagesInformation, deepLinkParams?: Partial<IntegrationParams>) {
```

Comment:
In generateSteps, do we need to know of the deeplinkparams since they are already  reflected in the existingConfiguration param? Would it be better if we pass a boolean instead of whether deeplinking is enabled or not instead, then   checking if `isDeeplinkingEnabled `&& state.pluginName then skip prompt?